hack/testing/test-upgrade.sh

```diff
@@ -233,7 +274,10 @@ function verifyUpgrade() {
       [[ -n "$(oc logs $UPGRADE_POD | grep 'Migration skipped for project '$project' - using common data model')" ]] && continue
       [[ -z "$(oc logs $UPGRADE_POD | grep 'Migration for project '$project': {"acknowledged":true}')" ]] && return 1
     done
+  fi
 
+  if [ $checkCDMMigrate = true ]; then
+    [[ -n "$(oc logs $UPGRADE_POD | grep 'Migration skipped for project oldindex.'${genUUID}'.'${indexDate}' - using common data model')" ]] && return 1
```

Comment:
should we also check that we can find ^{"acknowledged": true} in the deployer logs?